src/System.Management.Automation/engine/Attributes.cs

```diff
@@ -1114,6 +1114,11 @@ public sealed class ValidatePatternAttribute : ValidateEnumeratedArgumentsAttrib
         public RegexOptions Options { set; get; } = RegexOptions.IgnoreCase;
 
         /// <summary>
+        /// Gets or sets the custom error message that is displayed to the user
+        /// </summary>
+        public string ErrorMessage { get; set; }
```

Comment:
If you can specify `ErrorMessage` and it *might* be ignored, that's poor UX because then you must teach people where it is honored.

I see a bunch of custom validation attributes in this query: https://github.com/search?q=ValidateArgumentsAttribute&type=Code&utf8=%E2%9C%93

Based on the number of interesting hits, I think it would be a mistake to add `ErrorMessage` to `ValidateArgumentsAttribute`.